### ISSUE TITLE
Accessibility issues version 4.9.5 in relation to WCAG 2.1 conformity

### ISSUE THREAD

**Issue by larrykind** [globaleaks/GlobaLeaks#3222]
We noted some accessibility problems, reference is [https://www.w3.org/TR/WCAG21](https://www.w3.org/TR/WCAG21).
- 4.1.1 There are duplicate IDs (on some buttons and fields)
- 4.1.2 There are some buttons without text or label
- 4.1.2 Some of the form fields don't have linked `<label>` (pointing id where are the labels are missing)
- _Best practices_: the website seems to not use headers, so it's not possible to distinguish a title from normal text. (there are not "H" elements) 

 - OS: ubuntu 18 bionic
 - Browser: chrome 100, FF 100
 - GL Version: 4.9.5

**Comment by evilaliv3** [comment]
Thank you @larrykind.

The suggestion are surely good but miss proper detail. If you can please list the pages that have issues and the suggestion that you think should be applied; Alternatively if you have the possibility i invite you to perform a pull request directly to propose your changes.

**Comment by larrykind** [comment]
@evilaliv3 below follows a more detailed analysis:
-------------------------------------------------------------------------
### Accessibility Issues

#### 4.1.1 Duplicate ID elements are present
Some elements have the same 
For example, the id `#step-0-field-2-0-input-0` is repeated not less than 7 times in the tab 'Report'.
The ids should be unique

#### 4.1.2 There are buttons without text or a label
Some examples:
 - On the "Send Report" modal, the "X" button has no descriptive text, so it is impossible for speech synthesis assistants to understand its meaning
 - The same applies to the '+' button for adding 'Subjects involved in the facts'.

Therefore ensure that all buttons and interactive elements have descriptive text

#### 4.1.2 Some form fields do not have `<label>` attached 
All labels in the form have `<label>` without the `<id>` on the reference field

#### Best practices: The site does not use headings, so it is impossible to identify a title from the normal text It is therefore suggested to use tags for each title in the application.
---------------------------------------------------------------------------
Please tell me if you need something more specific

**Comment by evilaliv3** [comment]
Thank you @larrykind.

The input is detail is helpful and perfect and i think this corrections could be issued right away.

Would you please clarify about the tools that you are using to validate the globaleaks accessibility? This would be helpful in order to validate the correction. Thank you if you could share.

**Comment by evilaliv3** [comment]
@larrykind: I should have fixed all the reported issues and the corrections are going to be included in release 4.9.6

**Comment by larrykind** [comment]
Thanks @evilaliv3, 
the tool we used to this first accessibility check is [Siteimprove](https://www.siteimprove.com/), using browser plugin.
Wi didn't test with a screen reader yet but we will do soon and give you a feedback.
